core/src/main/java/edu/ucr/cs/riple/core/explorers/Explorer.java

```diff
@@ -41,16 +41,19 @@ public abstract class Explorer {
   protected final Bank<Fix> fixBank;
   protected final ImmutableSet<Report> reports;
   protected final Config config;
-  protected final FixGraph<Node> fixGraph;
+  protected final ConflictGraph graph;
 
   protected final MethodInheritanceTree methodInheritanceTree;
 
+  protected final int depth;
+
   public Explorer(
       AnnotationInjector injector,
       Bank<Error> errorBank,
       Bank<Fix> fixBank,
       ImmutableSet<Fix> fixes,
       MethodInheritanceTree methodInheritanceTree,
+      int depth,
```

Comment:
Since we are doing the analysis on downstream modules to only depth 1 and is different from the actual target module. Please let me know if you prefer changing this.